src/components/nav.tsx

```diff
@@ -138,6 +138,15 @@ export function Nav({ isCollapsed, links, supportLinks }: NavProps) {
                 <p className="text-xs">{user.primaryEmailAddress?.emailAddress}</p>
               </div>
             </div>
+            )}
+            {isCollapsed && (
+            <div className="flex items-center gap-2 ml-[3.5px]">
```

Comment:
@vcristian1 maybe instead of using a hard-coded margin here, we can fix the spacing issues my making the avatar size smaller. So that it works with the natural composition of flex box.

Reply:
Good point. I just did a git pull on new-stuff, removed the margin class in nav.tsx, adjusted the height and width in avatar.tsx to 12, and commited the changes.